Set the working dir (TEMPDIR) to /var/tmp/havp instead of /var/tmp

diff -Nur havp-0.92a.orig/etc/havp/havp.config.in havp-0.92a/etc/havp/havp.config.in
--- havp-0.92a.orig/etc/havp/havp.config.in	2011-12-20 22:08:18.907739050 -0600
+++ havp-0.92a/etc/havp/havp.config.in	2011-12-20 22:08:55.267259030 -0600
@@ -120,7 +120,7 @@
 # Needs to be writable by HAVP user. Use ramdisk for best performance.
 #
 # Default:
-# TEMPDIR /var/tmp
+# TEMPDIR /var/tmp/havp
 
 #
 # HAVP reloads scanners virus database by receiving a signal
diff -Nur havp-0.92a.orig/havp/params.cpp havp-0.92a/havp/params.cpp
--- havp-0.92a.orig/havp/params.cpp	2010-04-02 08:32:42.000000000 -0500
+++ havp-0.92a/havp/params.cpp	2011-12-20 22:08:44.061406971 -0600
@@ -70,7 +70,7 @@
     SetConfig("WHITELIST",	WHITELISTFILE);
     SetConfig("BLACKLIST",	BLACKLISTFILE);
     SetConfig("TEMPLATEPATH",	TEMPLATEPATH);
-    SetConfig("TEMPDIR",	"/var/tmp");
+    SetConfig("TEMPDIR",	"/var/tmp/havp");
     SetConfig("SCANTEMPFILE",	"/var/tmp/havp/havp-XXXXXX");
     SetConfig("PIDFILE",	PIDFILE);
     SetConfig("TRANSPARENT",	"false");
